### DISCUSSION TITLE
Pull Shark X3

### DISCUSSION THREAD

**Discussion by kirill-katkov** [community/community#47755] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

GitHub Profile Achievements
https://github.com/Schweinepriester/github-profile-achievements
Hello! 128 pull requests merged but I never got the badge x3.
Tell me what is the reason?
Thank you in advance

<img width="532" alt="Снимок экрана 2023-02-17 в 13 39 54" src="https://user-images.githubusercontent.com/95480051/219621791-fe307ad3-49a5-4838-a6ab-0e7d4b808a38.png">

<img width="326" alt="Снимок экрана 2023-02-17 в 13 41 21" src="https://user-images.githubusercontent.com/95480051/219622109-30c42dae-3586-47b7-9658-fcee3fb08fea.png">

**Comment by diannaowa** [comment]
I'm also facing the same problem

**Comment by radar-blip** [comment]
@kirill-katkov You have gotten it now. Maybe it just took some time.

**Comment by darkmastermindz** [comment]
@kirill-katkov @diannaowa @linuxlawson looks like x3 is for over 128 PRs according to  #18248.

**Comment by drippy-cat** [comment]
The issue should be resolved, it appears that you have gotten the badge now. Badges usually take a few minutes to update.

**Comment by YasinDehfuli** [comment]
** Don't worry, After 129 pull request you will get X3 Achievements.**

if You want to get other achievements and you need help, You can use this link to learn how to get other achievements step by step:

https://github.com/4xmen/Get-Github-Achievements
